huxley/www/js/text/DelegateCommitteeFeedbackViewText.md

```diff
@@ -0,0 +1,3 @@
+Thank you for attending BMUN {{ conferenceSession }} {{ firstName }}!  Please leave your feedback for **{{ committee }}**. 
+
+All delegate feedback is anonymous.
```

Comment:
Add something here in bold to stress they get to submit only once

Reply:
Done